api/routes/apds/post.js

```diff
@@ -12,6 +19,12 @@ module.exports = (app, ApdModel = defaultApdModel) => {
         state_id: req.user.state,
         status: 'draft'
       });
+
+      const state = await StateModel.where({ id: req.user.state }).fetch();
+      if (state.get('medicaid_office')) {
+        newApd.set({ stateProfile: state.get('medicaid_office') });
+      }
```

Comment:
Using the `medicaid_office` property on the state model as the full state profile for now.  Probably ought to clean up the model, but that touches a lot more places so I left it alone for now.